continuous_queries.md

```diff
@@ -0,0 +1,236 @@
+# Continuous Queries
+
+This document lays out continuous queries and a proposed architecture for how they'll work within an InfluxDB cluster.
+
+## Definition of Continuous Queries
+
+Continuous queries serve two purposes in InfluxDB:
+
+1. Combining many series into a single series (i.e. removing 1 or more tag dimensions to make queries more efficient)
+2. Aggregating and downsampling series
+
+The purpose of both types of continuous queries is to duplicate or downsample data automatically in the background to make querying thier results fast and efficient. Think of them as another way to create indexes on data.
+
+Generally, there are continuous queries that create copyies of data into another measurement or tagset and queries that downsample and aggregate data. The only difference between the two types is if the query has a `GROUP BY time` clause.
+
+Before we get to the continuous query examples, we need to define the `INTO` syntax of queries.
+
+### INTO
+
+`INTO` is a method for running a query and having it output into either another measurement name, retention policy, or database. The syntax looks like this:
+
+```sql
+SELECT *
+INTO [<retention policy>.]<measurement> [ON <database>]
```

Comment:
@pauldix can we choose a different separator for `<retention policy>.<measurement>`?  Otherwise, the parser can't distinguish between that and a measurement name with one or more '.' characters in it.